apps/web/src/pages/Profile/AccountSettings/AccountSettingsPage.tsx

```diff
@@ -0,0 +1,259 @@
+                  ))}
+                </TableOfContents.List>
+              </TableOfContents.Navigation>
+              <TableOfContents.Content>
+                <TableOfContents.Section
+                  id={sections.accountManagement.id}
+                  data-h2-padding-top="base(x3)"
+                >
+                  <TableOfContents.Heading
+                    size="h3"
+                    icon={Cog8ToothIcon}
+                    color="primary"
+                    data-h2-margin="base(0, 0, x1, 0)"
+                  >
+                    {sections.accountManagement.title}
+                  </TableOfContents.Heading>
+                  <p data-h2-margin="base(0, 0, x1, 0)">
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "This section focuses on general account management and information related to how we link to your GC Key.",
+                      id: "FmfkY6",
+                      description:
+                        "Subtitle for account management section on account settings page.",
+                    })}
+                  </p>
+                  <AccountManagement />
+                </TableOfContents.Section>
+                <TableOfContents.Section
+                  id={sections.notificationSettings.id}
+                  data-h2-padding-top="base(x3)"
+                >
+                  <TableOfContents.Heading
+                    size="h3"
+                    icon={Cog8ToothIcon}
+                    color="primary"
+                    data-h2-margin="base(0, 0, x1, 0)"
+                  >
+                    {sections.notificationSettings.title}
+                  </TableOfContents.Heading>
+                  <p data-h2-margin="base(0, 0, x1, 0)">
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "The settings provided in this section allow you to control when you receive notifications and how they are delivered. Email notifications are delivered to the email provided on your profile, while in-app notifications are delivered to the notification pane found in the main menu next to your name.",
+                      id: "hnyf1f",
+                      description:
+                        "Subtitle for notification settings section on account settings page.",
+                    })}
+                  </p>
+                  <NotificationSettings
+                    ignoredEmailNotifications={ignoredEmailNotifications}
+                    ignoredInAppNotifications={ignoredInAppNotifications}
+                  />
+                </TableOfContents.Section>
+                <TableOfContents.Section
+                  id={sections.recruitmentAvailability.id}
+                  data-h2-padding-top="base(x3)"
+                >
+                  <TableOfContents.Heading
+                    size="h3"
+                    icon={IdentificationIcon}
+                    color="primary"
```

Comment:
This is red in the design, is this intended?

Reply:
Nice catch! Fixed in [0e2a026](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/0e2a026ced61cd2337f7d8b6a9007f8470db6928)